python/cuml/tests/dask/test_dask_ordinal_encoder.py

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) 2020-2023, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import dask_cudf
+import numpy as np
+import pandas as pd
+import pytest
+from cudf import DataFrame
+from cuml.dask.preprocessing import OrdinalEncoder
+from distributed import Client
+
+
+@pytest.mark.mg
+def test_ordinal_encoder_df(client: Client) -> None:
```

Comment:
Can we add a corresponding `test_ordinal_encoder_array()` test function, please?